test/tempfile_test.py

```diff
@@ -0,0 +1,43 @@
+import os
+import unittest
+import ray
+
+
+class TempFileTest(unittest.TestCase):
```

Comment:
Sorry about this, but all new tests should use `pytest`. We're almost done converting all of the old tests to pytest.

Also, I'm not sure this warrants a whole new test file.

Reply:
I think it's ok to have it as a separate file. I'm not sure what the best way to organize the tests is.